dynatrace/environment_v2/service_level_objectives.py

```diff
@@ -0,0 +1,282 @@
+
+        :returns Response: HTTP response for the request
+        """
+        params = {
+            "name": name,
+            "target": target,
+            "warning": warning,
+            "timeframe": timeframe,
+            "useRateMetric": use_rate_metric,
+            "metricRate": metric_rate,
+            "metricNumerator": metric_numerator,
+            "metricDenominator": metric_denominator,
+            "evaluationType": evaluation_type,
+            "filter": filter_,
+            "customDescription": custom_description,
+            "enabled": enabled,
+        }
+        return self.__http_client.make_request(path=self.ENDPOINT, method="POST", params=params)
+
+    def update(
+        self,
+        slo_id: str,
+        name: Optional[str] = None,
+        target: Optional[float] = None,
+        warning: Optional[float] = None,
+        timeframe: Optional[str] = None,
+        use_rate_metric: Optional[bool] = None,
+        metric_rate: Optional[str] = None,
+        metric_numerator: Optional[str] = None,
+        metric_denominator: Optional[str] = None,
+        evaluation_type: Optional[str] = None,
+        filter_: Optional[str] = None,
+        custom_description: Optional[str] = None,
+        enabled: Optional[bool] = None,
+    ) -> "Response":
+        """Creates a new SLO.
+
+        :param slo_id: The ID of the existing SLO.
+        :param name: The name of the SLO.
+        :param target: The target value of the SLO.
+        :param warning: The warning value of the SLO. At warning state the SLO is still fulfilled but is getting close to failure.
+        :param timeframe: The timeframe for the SLO evaluation. Use the syntax of the global timeframe selector.
+        :param use_rate_metric: The type of the metric to use for SLO calculation - an existing percentage-based metric (true) or a ratio of two metrics (false)
+        :param metric_rate: The percentage-based metric for the calculation of the SLO. Required when the useRateMetric is set to true.
+        :param metric_numerator: The metric for the count of successes (the numerator in rate calculation).Required when the useRateMetric is set to false.
+        :param metric_denominator: The total count metric (the denominator in rate calculation). Required when the useRateMetric is set to false.
+        :param evaluation_type: The evaluation type of the SLO.
+        :param filter_: The entity filter for the SLO evaluation. Use the syntax of entity selector.
+        :param custom_description: The custom description of the SLO.
+        :param enabled: The SLO is enabled (true) or disabled (false).
+
+        :returns Response: HTTP response for the request
+        """
+        slo = self.get(slo_id=slo_id)
+        params = slo._to_json()
+
+        for key, val in locals().items():
+            if key in ["self", "slo_id", "slo", "params"]:
+                continue
+            if val is not None:
+                key = arg_to_camelcase(key)
```

Comment:
This part is very confusing and using locals() and magically converting things to camelCase is not good for the rest of us, lets always be explicit even if it means more code

Reply:
Agreed. Got lazy.